### ISSUE TITLE
not able to start the project

### ISSUE THREAD

**Issue by hassnaseer** [nguyentung9820/DEX#7]

**Comment by nguyentung9820** [comment]
Please check your node version, nodejs version require 16.13.1

**Comment by hassnaseer** [comment]
yes, it is same . i have start the project by my  own now i got this issue that is didn't see the list of coins:
![image](https://user-images.githubusercontent.com/78788462/173830327-0752a600-4fb9-4664-85b3-4a60b2ad9aee.png)
